docs/backstage/config.jsonnet

```diff
@@ -187,9 +187,13 @@
             {
               id: 'tags',
               name: 'Tags',
-              type: 'String',
               array: true,
               source: 'metadata.tags',
+              enum: {
```

Comment:
This is the new behaviour enabled by this change, where you can say "this attribute should be an enum" and we'll automatically create a type with entries for each value in it.

For Backstage users, that will probably mean "this component is tagged language-ruby, let's click to see all other components tagged with language-ruby".